src/sora-economy.md

```diff
@@ -1,9 +1,124 @@
+
+The conclusions were that there is a large amount of empirical evidence that suggests that **it is the change in the allocation of money in an economy that is a necessary and sufficient determinant of growth**. This was shown by Richard Werner in his [disaggregated quantity theory of money](https://eprints.soton.ac.uk/36569/), where he disaggregated Fisher's [equation of exchange](https://en.wikipedia.org/wiki/Equation_of_exchange) (MV = PY; a good way to think about this is also that MV equals nominal GDP viewed from the perspective of buyers, whereas PY equates to nominal GDP viewed from the perspective of producers/sellers) into *real* (*R*) and *financial* (*F*) transactions:
+
+$$
+V_R\Delta M_R = \Delta(P_RY)
+$$
+
+$$
+V_F\Delta M_F = \Delta(P_FQ_F)
+$$
+
+Where:
+
+*V* := velocity of money
+
+*M* := quantity of money
+
+*P* := price index
+
+*Y* := output of goods/services
+
+*Q* := quantity demanded
+
+It is helpful to think of the above equations as a dynamical system, such that if the quantity of money, *M*,  changes on the left-hand side of the equation, there must be some change on the right-hand side to balance out this change (this is because *V* is typically fairly stable in large economies). **For transactions in the real economy, this means that new money put into circulation is balanced out by new goods and services that consume this new purchasing power.** However, for financial transactions, there are no new goods or services being created with the new money put into circulation, so this causes a rise in asset prices to compensate (which is not what you want in your economic system). Additionally, it is also important to understand the importance of expanding M for the production of new goods and services. As [Shimomura wrote about extensively](https://www.jstage.jst.go.jp/article/economics1950/11/3-4/11_3-4_1/_article/-char/ja/) and Kurihara distilled to a simple, [linear programming model](https://books.google.co.jp/books?id=RVtGAQAAQBAJ&pg=PA60&lpg=PA60&dq=Japan%E2%80%99s+Credit-Supported+Growth+and+Linear+Programming&source=bl&ots=dNNHw-QyMh&sig=ACfU3U0gv9WNMZbWJBlfDf7AL5PY7aZzTQ&hl=ja&sa=X&ved=2ahUKEwjV8rf2qfHoAhUQx4sBHf2cBaEQ6AEwAHoECAwQKw#v=onepage&q=Japan%E2%80%99s%20Credit-Supported%20Growth%20and%20Linear%20Programming&f=false), expanding the quantity of money for new goods and services expands the economic output because latent resources (especially human resources) are mobilized, and existing resources are upgraded by the creation of new capital for production.
```

Comment:
I'd break the paragraph
1) before " However, for financial transactions"
2) before "Additionally, it is also important"

Reply:
Added breaks